sdk/cognitivelanguage/Azure.AI.Language.QuestionAnswering/src/ShortAnswerOptions.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core;
+
+namespace Azure.AI.Language.QuestionAnswering
+{
+    public partial class ShortAnswerOptions
```

Comment:
Yeah, I see it too. Maybe you were looking at the ShortAnswerOptions.Serialization.cs file? But as @pakrym pointed out, this would, at the very least, generate doc warnings and there are (almost) none:

<img width="300" alt="image" src="https://user-images.githubusercontent.com/1532486/139486816-e2637348-13e5-4075-9ea5-e3acbc655ba8.png">

Reply:
Apologies; that's my bad.   I didn't realize there were two generated partials and referenced the first in my list, which was indeed the serialization bit.